src/cc/libbpf.c

```diff
@@ -336,14 +338,18 @@ static int bpf_attach_tracing_event(int progfd, const char *event_path,
   return 0;
 }
 
-static void * bpf_attach_probe(int progfd, const char *event,
-                               const char *event_desc, const char *event_type,
-                               pid_t pid, int cpu, int group_fd,
-                               perf_reader_cb cb, void *cb_cookie) {
+void * bpf_attach_kprobe(int progfd, int attach_type, const char *ev_name,
+                        const char *fn_name,
+                        pid_t pid, int cpu, int group_fd,
+                        perf_reader_cb cb, void *cb_cookie) 
+{
   int kfd;
-  char buf[256];
+  char buf[strlen(ev_name)+256];
```

Comment:
i'm not sure what this 'strlen(ev_name)+256' math suppose to do.
if i understand it correctly, the size of this array is still a rough guess of how much buffer space is needed.
if so, just statically set it to 256. all ev_names suppose to be short?